tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs

```diff
@@ -260,9 +260,10 @@ public static HttpResponseMessage CreateFailureTokenResponseMessage(
             string error,
             string subError = null,
             string correlationId = null,
-            HttpStatusCode? customStatusCode = null)
+            HttpStatusCode? customStatusCode = null,
+            string errorCode = "AADSTS00000")
         {
-            string message = "{\"error\":\"" + error + "\",\"error_description\":\"AADSTS00000: Error for test." +
+            string message = "{\"error\":\"" + error + "\",\"error_description\":\"" + errorCode + ": Error for test." +

```

Comment:
Do you think it's worth it to use string interpolation on only this line? It might be "cleaner".

Reply:
I checked with the ai and it was about the same in complexity